src/Optimizer/Transformer/ServerSideRendering.php

```diff
@@ -1060,4 +1061,28 @@ private function isZero($number)
 
         return abs($number) < self::FLOATING_POINT_EPSILON;
     }
+
+    /**
+     * Server-side rendering of an amp-audio element.
+     *
+     * @param Document $document DOM document to apply the transformations to.
+     * @param Element  $element  Element to adapt.
+     */
+    private function ssrAmpAudio(Document $document, Element $element)
+    {
+        // Check if we already have a SSR-ed audio element.
+        if ($element->hasChildNodes()) {
+            foreach ($element->childNodes as $childNode) {
+                if ($childNode instanceof Element && $childNode->tagName === Tag::AUDIO) {
+                    return;
+                }
+            }
+        }
+
+        $audio = $document->createElement(Tag::AUDIO);
+        $audio->setAttribute(Attribute::CONTROLS, '');
```

Comment:
This should generate a boolean attribute rather than `controls=""`:

```suggestion
        $audio->setAttribute($document->createAttributeNode(Attribute::CONTROLS));
```